web/src/portico/hello.js

```diff
@@ -0,0 +1,55 @@
+(function () {
+    "use strict";
```

Comment:
@timabbott linter want me to write the code like this. Is that ok?

Reply:
No, make this an ES module and write an empty `export {}` declaration to mark it as such for webpack and ESLint.

Reply:
Updated!